src/components/music-list/music-list.style.jsx

```diff
@@ -0,0 +1,35 @@
+import { Table } from "antd";
+import styled from "styled-components";
+
+export const MusicListContainerStyled = styled('div')`
+  padding: 32px 160px 10px 160px;
+  background-color: #F8F8F8;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+`
+
+export const FilterTitleStyled = styled('p')`
+  color: #353D58;
+  font-size: 24px;
+`
+
+export const TableStyled = styled(Table)`
```

Comment:
Seria legal evitar o uso do !important, já que ele eh um antipattern

https://dev.to/alvaromontoro/using-important-in-css-75c#:~:text=here%20it%20goes%3A,difficult%20to%20maintain%20and%20debug.